reflog.txt

```diff
@@ -0,0 +1,218 @@
+5361deb4 HEAD@{0}: commit: wip fixup
```

Comment:
same here: is this meant to be really committed?

Reply:
Checked in by mistake, will remove.